pkg/osx/pkg-scripts/postinstall

```diff
@@ -49,6 +49,33 @@ if [ ! -f "$CONFIG_DIR/minion" ]; then
     cp "$CONFIG_DIR/minion.dist" "$CONFIG_DIR/minion"
     echo "Config: Copied Successfully" >> "$TEMP_DIR/postinstall.txt"
 fi
+###############################################################################
+# Create symlinks to pyenv directories
+###############################################################################
+echo "Symlink: Creating symlinks to pyenv directories..." >> "$TEMP_DIR/postinstall.txt"
+ln -s "$INSTALL_DIR/.pyenv/versions/3.7.12/bin" "$INSTALL_DIR/bin"
+ln -s "$INSTALL_DIR/.pyenv/versions/3.7.12/include" "$INSTALL_DIR/include"
+ln -s "$INSTALL_DIR/.pyenv/versions/3.7.12/lib" "$INSTALL_DIR/lib"
+
+echo "Symlink: Creating symlinks to library files..." >> "$TEMP_DIR/postinstall.txt"
+find "$INSTALL_DIR/.pyenv/versions/3.7.12" \
```

Comment:
There are a lot of paths in this PR that hardcode `3.7.12` . Can we use the `PY_DOT_VERSION` variable instead?